docs/reference/cli.md

```diff
@@ -199,18 +199,60 @@ target:  exists
 
 ## `if-env`
 
-The `if-env` command line tool allows you to create the environment in which the manifest can run.
+`if-env` is a command line tool that helps you to create local development environments where you can run manifests.
 
-`if-env` can be run without any arguments, it will create a template manifest file and a relevant package.json in the same directory where the command is run. By providing `--install` option, it will install the package.json.
+There are two use cases for this:
 
-`if-env` can be run as follows:
+1) setting up a new development environment for plugin building
+2) replicating a runtime environment for a given manifest, so you can re-execute it
+
+### commands
+
+- `--manifest` or `-m`: the path to a manifest whose dependencies you want to install
+- `--install` or `-i`: instructs `if-env` to automatically install the dependencies in the local `package.json`
+- `--cwd` or `-c`: forces `if-env` to create or update the package.json in the current working directory. This is already default behaviour when no arguments are passed to `if-env`, but when a manifest is passed to `-m`, `if-env` defaults to saving a package.json in the same folder as the manifest. using `-cwd` overrides that behaviour and uses the current working directory as the `package.json` target path.
+
+### Setting up new development environments using `if-env`
+
+If you are creating a new manifest from scratch and want to bootstrap your way in, you can use `if-env` with no arguments to generate a template manifest and packag.json in your current working directory. Then, all you need to do is tweak the templates for your specific use case. 
+
+For example:
+
+```sh
+mkdir my-manifest && cd my-manifest
+if-env
+```
+
+After running these commands, you will see the following files in `my-manifest`:
+
+```
+ls my-manifest
+
+> package.json manifest.yaml
+```
+
+Now, you can use these files as templates for your manifest development.
+
+
+### Replicating runtime environments using `if-env`
+
+If you are given an IF output file and you want to rerun it, you can use `if-env` to install that output file's dependencies so that all the plugins in its execution pipeline can be executed.
```

Comment:
```suggestion
If you are given an IF output file and you want to return it, you can use `if-env` to install that output file's dependencies so that all the plugins in its execution pipeline can be executed.
```